coral/src/domain/environment/environment-types.ts

```diff
@@ -14,13 +15,17 @@ import { KlawApiModel } from "types/utils";
 type Environment = {
   name: KlawApiModel<"EnvModelResponse">["name"];
   id: KlawApiModel<"EnvModelResponse">["id"];
-  defaultPartitions: number | undefined;
-  defaultReplicationFactor: number | undefined;
-  maxPartitions: number | undefined;
-  maxReplicationFactor: number | undefined;
-  topicNamePrefix: string | undefined;
-  topicNameSuffix: string | undefined;
   type: KlawApiModel<"EnvModelResponse">["type"];
+  params: {
+    // will be changed to Integer in BE at some point
+    // and we need it best as a number
+    defaultPartitions?: number;
+    defaultRepFactor?: number; // before: defaultReplicationFactor
+    maxPartitions?: number;
+    maxRepFactor?: number; // before: maxReplicationFactor
```

Comment:
Do we need the "before" comments? Will we need to know this information in the future?

Reply:
Oh, no, they were for my refactoring, will remove!